<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kindnew</class>
 <widget class="QDialog" name="Kindnew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Set category:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_Category"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Set gender:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_Gender"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Enter kind:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="add_dresskind">
     <property name="text">
      <string>Add kind</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_next">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
